Week2/homework/index.js

```diff
@@ -0,0 +1,114 @@
+console.log(1);
+let HiIndo='Hallo, dunia!'
+let HiIta='Ciao, mondo!' 
+let HiSpa='Hola, mundo!'
+let HiRus='Priviet, mir!'
+let HiKaz='Salamtsizba, duniye!'
+let HiDutch='Hallo, wereld!'
+let HiFr='Bonjour, le monde!'
+let HiTr='Merhaba, dunya!'
+let HiGr='Hallo, welt!'
+let HiRomanian='Salut, lume!'
+console.log(HiIndo);
+console.log(HiIta);
+console.log(HiSpa);
+console.log(HiRus);
+console.log(HiKaz);
+console.log(HiDutch);
+console.log(HiFr);
+console.log(HiTr);
+console.log(HiGr);
+console.log(HiRomanian);
+
+console.log(2);
+console.log('I'm awesome');
+// Uncaught SyntaxError: missing ) after argument list
+//The symbol used for the string is the same with the symbol used in the syntax of the string.
+// When I changed it into sth else, i gives what i wanted to write.
+console.log(" I'm awesome!");
+
+
+console.log(3)
+let x;
+x=3;
+
+console.log("the value of my variable x will be: 3");
+console.log(x);
+
+
+console.log(4);
+var y="I LOve Spring";
+console.log("The value of my variable y will be: I Love Spring!);
+
+var y="The Rain Is Beautiful!";
+console.log("the value of my variable y will be: The Rain Is Beautiful!");
+console.log(y);
+
+
+console.log('5th question');
+var z = 7.25;
+console.log(z);
+var a = Math.round(7.25);
+console.log(a);
+
+if (a > z) {let c = a;
+console.log(c)} else {let c = z; console.log(c)};
```

Comment:
Ahh, yeah this works technically! But an easier solution is using [Math.max](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max)